include/srtp.h

```diff
@@ -1424,7 +1408,7 @@ typedef enum {
 
 typedef struct srtp_event_data_t {
   srtp_t        session;  /**< The session in which the event happend. */
-  srtp_stream_t stream;   /**< The stream in which the event happend.  */
+  uintptr_t     ssrc;     /**< The ssrc in host order of the stream in which the event happend */
```

Comment:
Would anyone really be relying on the length of this structure, but not looking at the data?  I personally think it's a bad idea to assume the length of any data structure doesn't change.